src/modules/integrator-guide/pages/server-plugins/index.adoc

```diff
@@ -0,0 +1,113 @@
+include::ROOT:partial$variables.adoc[]
+
+= Server plugins
+
+{PRODUCT_NAME} implements the support for https://pkg.go.dev/plugin[Go plugins] which can be used to modify default behaviour of the server and provide new functionalities like workflow functions and types.
+
+== Basic plugin structure
+
+[source,go]
+----
+package main
+
+type (
+	examplePlugin struct {} <1>
+)
+
+func CortezaPlugin() interface{} { <2>
+	return &examplePlugin{}
+}
+----
+<1> Struct that holds
+<2> Be sure you return an `interface{}`, otherwise {PRODUCT_NAME} can not cast it to plugin
```

Comment:
End both lines with a dot

If you're defining a list where the line items don't formulate a proper sentence, those are ended with a comma instead of a line -- "If the list item is short, end with a comma; if the list item is long, end with a dot"